ta/os_test/Makefile

```diff
@@ -2,14 +2,29 @@ include $(TA_DEV_KIT_DIR)/mk/conf.mk
 
 BINARY = 5b9e0e40-2636-11e1-ad9e-0002a5d5c51b
 
+OS_TEST_LIB = ""
+OS_TEST_LIB_LDADD = ""
+
 ifeq ($(CFG_TA_DYNLINK),y)
 # Hack: to locate the os_test library, we assume that its output path
 # follows the same scheme as the output path of the os_test application.
 # Therefore, simply replacing os_test by os_test_lib in the absolute path
 # should get us a valid path
-LDADD = -L$(subst os_test,os_test_lib,$(abspath $(link-out-dir))) -los_test
+OS_TEST_LIB_LDADD = -L$(subst os_test,os_test_lib,$(abspath $O)) -los_test
+OS_TEST_LIB = os_test_lib
 endif
 
 LDADD += -ldl
 
-include ../ta_common.mk
+# ensure os_test_lib is present
+.PHONY: os_test_lib
+os_test_lib:
+	@echo .. suspended while building $(subst os_test,os_test_lib,$(abspath $(@D)))/Makefile
```

Comment:
This line is not needed since make outputs messages already:
```
make[3]: Entering directory '<...>/optee_test/ta/os_test'
make[4]: Entering directory '<...>/optee_test/ta/os_test_lib'
[...]
make[4]: Entering directory '<...>/optee_test/ta/os_test_lib'
make[3]: Entering directory '<...>/optee_test/ta/os_test'